config/stack/manifests/resources/cockroachcluster.resource.yaml

```diff
@@ -0,0 +1,35 @@
+id: cockroachcluster
+title: CockroachDB Cluster
+titlePlural: CockroachDB Clusters
+category: Database
+overviewShort: "A CockroachCluster configures a Rook 'CockroachCluster'"
```

Comment:
According to https://rook.io/docs/rook/v1.1/cockroachdb-cluster-crd.html, the actual kind of the Rook cockroach CRD is `Cluster`.  If we want to use the fully qualified type it's `clusters.cockroachdb.rook.io`